generators/client/templates/angular/src/main/webapp/app/app-routing.module.ts.ejs

```diff
@@ -44,7 +44,7 @@ const LAYOUT_ROUTES = [
                 <%_ if (!skipUserManagement) { _%>
                 {
                     path: 'account',
-                    loadChildren: () => import('./account/account.module').then(m => m.<%=angularXAppName%>AccountModule)
```

Comment:
Are you sure about this ? I think we added a prefix to handle some conflicts, maybe when generating entities... What if I generate an entity named `Account` ?

Reply:
If trying to generate entity with name `Account` then entity sub generator throws error "ERROR! Error: The entity name cannot contain a Java or JHipster reserved keyword".

There is issue #10220 opened which contains also prefix removing and some prefixes are already removed. I put reference to this issue in previous commit but as this is in the second line then GitHub doesn't show this here (only first line of the commit message is shown).